tests/unit/amo/components/TestFeaturedAddons.js

```diff
@@ -180,14 +180,13 @@ describe('<FeaturedAddons />', () => {
         visibleAddonType: visibleAddonType(addonType),
       },
       ...mappedProps,
-      hasSearchParams: true,
     });
 
     dispatch.reset();
 
     root.setProps({
       // Update an unrelated parameter.
-      hasSearchParams: false,
+      something: false,
```

Comment:
I like your approach in the other test better (which I didn't think of in this one, heh):

```js
root.setProps();
```

I think updating a non-existent property isn't helpful because it could produce an error one day if we get strict about validating props.